gui/system/forms.py

```diff
@@ -923,11 +925,20 @@ def clean(self):
         if proto == "http":
             return cdata
 
-        certificate = cdata["stg_guicertificate"]
-        if not certificate:
+        if not cdata["stg_guicertificate"]:
             raise forms.ValidationError(
                 "HTTPS specified without certificate")
-
+        else:
+            certificate_obj = models.Certificate.objects.get(cert_name=cdata["stg_guicertificate"])
+            fingerprint = certificate_obj.get_fingerprint()
+            # We need to write this to the system console (i.e. /dev/ttyv0)
+            # see ticket: #6605 for more ifo as to why
+            with open('/dev/ttyv0', 'wb') as f:
```

Comment:
I am not sure directly opening ttyv0 is a good idea? Is thre a good reason for this?
log.error would log to syslog and console at the same time.